core/interfaces/i_registrable_field.js

```diff
@@ -11,21 +11,25 @@
 
 'use strict';
 
-goog.provide('Blockly.IRegistrableField');
+goog.module('Blockly.IRegistrableField');
+goog.module.declareLegacyNamespace();
 
 goog.requireType('Blockly.Field');
 
+
```

Comment:
Whoop whoop!

Reply:
I added the newline here because all but 3 of our files use 2 newlines after our requires except for:
- `core/interfaces/i_registrable_field.js`
- `core/delete_area.js`
- `core/input_types.js`

For that reason I think it makes sense to add the newline here for consistency.

Reply:
Yeah, nice.